features/networking/multus.feature

```diff
@@ -464,9 +469,9 @@ Feature: Multus-CNI related scenarios
     #Entering into corresponding no eot make sure No VLAN ID information shown for secondary interface
     Given CNI vlan info is obtained on the "<%= cb.pod.node_name %>" node
     Then the step should succeed
-    And the output should contain 2 times:
-      | 1 PVID   |
-      | ntagged |
+    And evaluation of `@result[:parsed]` is stored in the :vlans_after clipboard
+    Given the number of bridge PVID 1 "Egress Untagged" VLANs added between the :vlans_before and :vlans_after clipboards is 2
+
```

Comment:
new step

Reply:
Thanks , Ross. We should strictly check ```ntagged```  only to make compatibility across various platforms as per reasoning in other comment. Can you also pastebin logs? Seems like that one got are empty.